cli/src/build.rs

```diff
@@ -1,4 +1,7 @@
-use crate::style::{self, blue_bold, print_green_bold};
+use crate::{
```

Comment:
In my rust files I put 1 import per line to prevent merge conflicts. I order the imports as `std` first, external crates second, and local modules third. A good example is [here](https://github.com/Neptune-Crypto/twenty-first/blob/master/twenty-first/src/amount/u32s.rs#L26)

Reply:
> In my rust files I put 1 import per line to prevent merge conflicts. I order the imports as `std` first, external crates second, and local modules third. A good example is [here](https://github.com/Neptune-Crypto/twenty-first/blob/master/twenty-first/src/amount/u32s.rs#L26)

I follow this rule as well. we could consider enabling [group_imports](https://rust-lang.github.io/rustfmt/?version=v1.4.38&search=#group_imports) in **rustfmt** to help avoid conflicts and keep lines shorter.
It’s an unstable feature but might be worth trying.

Reply:
@sifnoc Yes currently I don't follow the order to import packages
We can do that in another PR 👍🏻